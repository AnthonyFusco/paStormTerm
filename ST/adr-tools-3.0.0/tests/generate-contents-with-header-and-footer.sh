adr new First Decision
adr new Second Decision
adr new Third Decision
cat > intro.md <<EOF
An intro.

Multiple paragraphs.
EOF
cat > outro.md <<EOF
An outro.
EOF
adr generate toc -i intro.md -o outro.md
