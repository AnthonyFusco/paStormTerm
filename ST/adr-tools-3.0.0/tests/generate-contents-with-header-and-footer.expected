adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr new Third Decision
doc/adr/0003-third-decision.md
cat > intro.md <<EOF
An intro.

Multiple paragraphs.
EOF
cat > outro.md <<EOF
An outro.
EOF
adr generate toc -i intro.md -o outro.md
# Architecture Decision Records

An intro.

Multiple paragraphs.

* [1. First Decision](0001-first-decision.md)
* [2. Second Decision](0002-second-decision.md)
* [3. Third Decision](0003-third-decision.md)

An outro.
